### PR TITLE
Feat: [OV-39584] Stepper initial release

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: packages/Stepper/src/Stepper.test.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    const currentStep = 1;
+    
+    const {container} = setup({steps: steps, currentStep: currentStep, clickableNextSteps: true});
+    const stepElements = container.querySelectorAll('.ids-step');
+    for (let i = currentStep + 1; i < stepElements.length; i++) {
+      expect(stepElements[i]).not.toBeDisabled();
+    }
+  });
+
+
```
Comment (ofrogon) [inline packages/Stepper/src/Stepper.test.tsx:87]: Too much blank lines 😋

File: packages/Stepper/src/step.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.ids-step__wrapper {
+  align-items: center;
+  display: flex;
+  flex: 0 0 auto;
+  justify-content: center;
+  min-width: 0;
+  position: relative;
+
```
Comment (ofrogon) [inline packages/Stepper/src/step.scss:31]:
```suggestion
```
(forgot a blank line)

File: packages/Stepper/src/step.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    flex: 0 0 auto;
+    height: var(--ids-step-current-dot-size);
+    width: var(--ids-step-current-dot-size);
+  }
+}
+
+.ids-step__checkmark {
+  flex: 0 0 auto;
+}
+
```
Comment (ofrogon) [inline packages/Stepper/src/step.scss:95]:
```suggestion
```
(blank line)

**Review by ofrogon — COMMENTED** [review]
Good job, only some minor changes to do

**Review by franckgaudin — APPROVED** [review]

**Review by ofrogon — APPROVED** [review]

**Pull request merged**